postgresql/setup.py

```diff
@@ -43,7 +43,7 @@ def get_changelog():
         "Programming Language :: Python :: Implementation :: PyPy",
         "Topic :: Internet :: WWW/HTTP :: WSGI :: Application"] + [
             "Programming Language :: Python :: %s" % x
-            for x in "3.4 3.5 3.6".split()],
+            for x in "2.7 3.4 3.5 3.6 3.7".split()],
```

Comment:
devpi-postgresql doesn't support Python 2.7 since 1.0.0. This was made deliberately to push people to 3.x.